semgrep-core/matching/Generic_vs_generic.ml

```diff
@@ -16,6 +16,7 @@
  *)
 (*e: pad/r2c copyright *)
 open Common
+open List
```

Comment:
We don't have any ocaml coding style with @mjambon yet, but I usually try to avoid to use 'open' and prefer to qualify functions used in other modules because it's harder then when you see just  'foo arg' to see to which module it belongs to. You can use IDEs to help, but it still IMHO makes the code slightly harder to read.

There are a few exceptions when it's just annoying to qualify the same module again and again. For example I usually do open AST_generic when I know a module will exclusively use entities defined in this AST.

Reply:
fixes issues in pfff together!